### DISCUSSION TITLE
Running HA-Fusion twice or with several dashboards

### DISCUSSION THREAD

**Discussion by Quaddel** [matt8707/ha-fusion#537] [Ideas]
Hello :)

I recently discovered HA-Fusion and I'm in love.

After configuring everything for my 22" touchscreen in the living room, I also wanted to configure the smaller 10" touchscreen on the upper floor.

Unfortunately, the resolution here is lower and I have to scroll all the time.

Wouldn't it be cool if you could run HA-Fusion as another independent instance?

Or optionally create another dashboard. So you could also, for example, create your own configurations for the smartphone :)

**Comment by jkrenzer** [comment]
Hi! 

Having several dashboards, probably addressable by url-path, would also allow to have dedicated ones for roomes or floors of the house. Running an independent instance for each might be not the way to go, as it would be difficult for folks using the addon in homeassistant. 

Cheers,

Jörn

**Comment by OlofBailey** [comment]
I'm after the same... looking to replace 15 crestron touch screens and want to make them each default to the screen for the room they are installed in.

**Comment by OlofBailey** [comment]
Looks like it's possible https://github.com/matt8707/ha-fusion?tab=readme-ov-file#query-strings

**Reply by jkrenzer** [reply]
> Looks like it's possible https://github.com/matt8707/ha-fusion?tab=readme-ov-file#query-strings

I don't think so. Views are the tabs visible in the dashboard on the right as I understand. So you can only choose which of the tabs is shown when opening the url, but everything else is still the same.

So it is not possible to have e.g. different sidebar entries or entities in the views.

**Comment by semyonchetvertnyh** [comment]
Of course it's possible. Just add the repository to Add-on store twice: with _http://_ and _https://_ at the beginning and configure different ports of the instances.

<img width="436" alt="Screenshot 2024-11-15 at 01 04 25" src="https://github.com/user-attachments/assets/dcde82f7-e36a-420f-935a-35bd36e35e5e">
